pointers/pointers.c

```diff
@@ -67,6 +84,8 @@ int string_compare(char *m, char *n)
 char *find_string(char *haystack, char *needle)
 {
 
+
+
 }
 
```

Comment:
You were having trouble with this part, right? What was the approach you were planning?